### ISSUE TITLE
The mod has broken with the new update. 

### ISSUE THREAD

**Issue by Drewbies** [BarleyFlour/WrathVisualAdjustmentsMod#159]
Is there an update coming soon? I know this isn't a real issue, but I don't know where I'd ask this question otherwise, the new update broke the mod.

**Comment by aivengo1677** [comment]
After 1.4 game update this mod is broken :(

`[VisualAdjustments] Version '1.13.8'. Loading.
[VisualAdjustments] HarmonyLib.HarmonyException: Patching exception in method null ---> System.NullReferenceException: Object reference not set to an instance of an object
  at HarmonyLib.PatchTools.GetOriginalMethod (HarmonyLib.HarmonyMethod attr) [0x00087] in <6ad64e60db9e43e5a66782c0bcf0cc7d>:0 
  at HarmonyLib.PatchClassProcessor.PatchWithAttributes (System.Reflection.MethodBase& lastOriginal) [0x0001c] in <6ad64e60db9e43e5a66782c0bcf0cc7d>:0 
  at HarmonyLib.PatchClassProcessor.Patch () [0x00059] in <6ad64e60db9e43e5a66782c0bcf0cc7d>:0 
   --- End of inner exception stack trace ---
  at HarmonyLib.PatchClassProcessor.ReportException (System.Exception exception, System.Reflection.MethodBase original) [0x00127] in <6ad64e60db9e43e5a66782c0bcf0cc7d>:0 
  at HarmonyLib.PatchClassProcessor.Patch () [0x00084] in <6ad64e60db9e43e5a66782c0bcf0cc7d>:0 
  at HarmonyLib.Harmony.<PatchAll>b__10_0 (System.Type type) [0x00007] in <6ad64e60db9e43e5a66782c0bcf0cc7d>:0 
  at HarmonyLib.CollectionExtensions.Do[T] (System.Collections.Generic.IEnumerable`1[T] sequence, System.Action`1[T] action) [0x00014] in <6ad64e60db9e43e5a66782c0bcf0cc7d>:0 
  at HarmonyLib.Harmony.PatchAll (System.Reflection.Assembly assembly) [0x00006] in <6ad64e60db9e43e5a66782c0bcf0cc7d>:0 
  at VisualAdjustments.Main.Load (UnityModManagerNet.UnityModManager+ModEntry modEntry) [0x00033] in <2428031b1be9447da5c231487a5f9c73>:0 
  at HarmonyLib.PatchClassProcessor.ReportException (System.Exception exception, System.Reflection.MethodBase original) [0x00127] in <6ad64e60db9e43e5a66782c0bcf0cc7d>:0 
  at HarmonyLib.PatchClassProcessor.Patch () [0x00084] in <6ad64e60db9e43e5a66782c0bcf0cc7d>:0 
  at HarmonyLib.Harmony.<PatchAll>b__10_0 (System.Type type) [0x00007] in <6ad64e60db9e43e5a66782c0bcf0cc7d>:0 
  at HarmonyLib.CollectionExtensions.Do[T] (System.Collections.Generic.IEnumerable`1[T] sequence, System.Action`1[T] action) [0x00014] in <6ad64e60db9e43e5a66782c0bcf0cc7d>:0 
  at HarmonyLib.Harmony.PatchAll (System.Reflection.Assembly assembly) [0x00006] in <6ad64e60db9e43e5a66782c0bcf0cc7d>:0 
  at VisualAdjustments.Main.Load (UnityModManagerNet.UnityModManager+ModEntry modEntry) [0x00033] in <2428031b1be9447da5c231487a5f9c73>:0 
[VisualAdjustments] [Error] Error trying to call 'VisualAdjustments.Main.Load'.
[VisualAdjustments] [Exception] TargetInvocationException - Exception has been thrown by the target of an invocation.
System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation. ---> HarmonyLib.HarmonyException: Patching exception in method null ---> System.NullReferenceException: Object reference not set to an instance of an object
  at HarmonyLib.PatchTools.GetOriginalMethod (HarmonyLib.HarmonyMethod attr) [0x00087] in <6ad64e60db9e43e5a66782c0bcf0cc7d>:0 
  at HarmonyLib.PatchClassProcessor.PatchWithAttributes (System.Reflection.MethodBase& lastOriginal) [0x0001c] in <6ad64e60db9e43e5a66782c0bcf0cc7d>:0 
  at HarmonyLib.PatchClassProcessor.Patch () [0x00059] in <6ad64e60db9e43e5a66782c0bcf0cc7d>:0 
   --- End of inner exception stack trace ---
  at VisualAdjustments.Main.Load (UnityModManagerNet.UnityModManager+ModEntry modEntry) [0x000e1] in <2428031b1be9447da5c231487a5f9c73>:0 
  at (wrapper managed-to-native) System.Reflection.MonoMethod.InternalInvoke(System.Reflection.MonoMethod,object,object[],System.Exception&)
  at System.Reflection.MonoMethod.Invoke (System.Object obj, System.Reflection.BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) [0x00032] in <695d1cc93cca45069c528c15c9fdd749>:0 
   --- End of inner exception stack trace ---
  at System.Reflection.MonoMethod.Invoke (System.Object obj, System.Reflection.BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) [0x00048] in <695d1cc93cca45069c528c15c9fdd749>:0 
  at System.Reflection.MethodBase.Invoke (System.Object obj, System.Object[] parameters) [0x00000] in <695d1cc93cca45069c528c15c9fdd749>:0 
  at UnityModManagerNet.UnityModManager+ModEntry.Invoke (System.String namespaceClassnameMethodname, System.Object& result, System.Object[] param, System.Type[] types) [0x00011] in <26aa0a937c05435e969262d957351813>:0 
[VisualAdjustments] Not loaded.`

**Comment by Kryzze** [comment]
Really hope this gets updated soon VA 2 is just not the same.

**Comment by Kryzze** [comment]
This is never getting updated is it :(

**Comment by psykolord** [comment]
> This is never getting updated is it :(

My assumption would be that if BarleyFlour _is_ planning to update, they'll be saving that until EE comes out fairly soon (next week I think?), _specifically_ because the devs allowed us (on steam at least) to opt in to the 1.3.7 version instead of going to 1.4.x. That's my _hope_ anyways, since the alternative is that this has been discontinued entirely in favor of VA2, and I'd really rather not be stuck switching mods.

**Comment by NakedGranny** [comment]
Mod no longer works at all with current versions of the game (as of EE release patch).

**Comment by Walters-Marissa** [comment]
I want this back, the new mod was a mistake.

**Comment by NakedGranny** [comment]
> I want this back, the new mod was a mistake.

Absolutely. VA2 just fucking sucks. It can't even do anything that people used VA for.

**Comment by MysticBrightShadow** [comment]
Please update!!! PLEASE!

**Comment by candleonaplate** [comment]
Dunno if the mod author is even reading this page anymore, but if you do, for the love of all that's holy, please update this and abandon VA2. It's a gigantic mess. Getting it to work is infuriatingly frustrating. VA1 worked just fine. The 'fancy' ui is useless if the mod cannot actually do what it advertises.
